### DISCUSSION TITLE
mac系统上，编译出来的QCefView库文件在哪里？

### DISCUSSION THREAD

**Discussion by HelloAllenZhu** [CefView/QCefView#207] [General]
mac系统上，编译出来的QCefView库文件在哪里？

**Comment by tishion** [comment]
没有framework么？

**Reply by HelloAllenZhu** [reply]
有一个Chromium Embedded Framework文件，但是QtCreator里面怎么使用这个文件呢？😂

**Reply by tishion** [reply]
是QCefView.framework，你可以了解一下qmake
https://stackoverflow.com/questions/3513907/linking-mac-framework-to-qt-creator

**Reply by HelloAllenZhu** [reply]
好的，谢谢！
